src/features/session_replay/shared/recorder.js

```diff
@@ -107,8 +107,9 @@ export class Recorder {
         if (failedToFix) {
           this.currentBufferTarget.inlinedAllStylesheets = false
           this.shouldFix = false
-          handle(SUPPORTABILITY_METRIC_CHANNEL, ['SessionReplay/Payload/Missing-Inline-Css/Failed', failedToFix], undefined, FEATURE_NAMES.metrics, this.parent.ee)
-        } else handle(SUPPORTABILITY_METRIC_CHANNEL, ['SessionReplay/Payload/Missing-Inline-Css/Fixed', incompletes - failedToFix], undefined, FEATURE_NAMES.metrics, this.parent.ee)
+        }
```

Comment:
`failedToFix` can't resolve to be undefined, so you can just remove this if-check.
Otherwise, both handles should go inside the if-check.

Reply:
<img width="287" alt="Screenshot 2024-02-23 at 8 52 18 AM" src="https://github.com/newrelic/newrelic-browser-agent/assets/24296585/d84fa479-07cf-4abe-b3d5-d5cb26328f1d">

Reply:
Then can you make these explicit:
`if (incompletes > 0) {`
`if (failedToFix > 0) {`